### PR TITLE
Add firewall policy data source

### PR DESCRIPTION
**Pull request by chrischdi**
This commit adds a data source for `openstack_fw_policy_v1` resources.
This is useful to refer to a policy ID on creation of a `openstack_fw_firewall_v1 `.

### CODE REVIEW

**Comment by chrischdi** [comment]
The Test:
```
Class:github.com/terraform-providers/terraform-provider-openstack/openstack
Test Name:TestAccComputeV2FloatingIPAssociate_attachToSecondNetwork
```
Did fail but I think this is not related to this PR?!

**Comment by jtopjian** [comment]
> Did fail but I think this is not related to this PR?!

Correct. This test occasionally fails and I think I'm just going to remove it since it's caused too many false failures.

**Comment by kiwik** [comment]
FWaaS patch still be in progress https://bugs.launchpad.net/neutron/+bug/1779978 , OpenLab fwaas job will pass until above patch merged. OpenLab job failing don't block PR merging, so feel free to bypass it.

File: openstack/data_source_openstack_fw_policy_v1.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+func dataSourceFWPolicyV1Read(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkingClient, err := config.networkingV2Client(GetRegion(d, config))
+
+	listOpts := policies.ListOpts{
+		ID:       d.Get("policy_id").(string),
+		Name:     d.Get("name").(string),
+		TenantID: d.Get("tenant_id").(string),
```
[the remark this answers is not in the document]
Comment (chrischdi) [reply openstack/data_source_openstack_fw_policy_v1.go]: Makes totally sense to me. I've added a commit to fix these values.
Thank you for your feedback!

**Comment by jtopjian** [comment]
@kiwik No problem - thanks for the update :smile:

**Pull request merged**